[2024-08-03T09:53:04.063+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-03T09:53:04.141+0000] {taskinstance.py:2066} INFO - Dependencies not met for <TaskInstance: etl_dag.Carga_Dimensao_user_type scheduled__2024-08-02T15:50:00+00:00 [upstream_failed]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=0, skipped=0, failed=1, upstream_failed=0, removed=0, done=1, success_setup=0, skipped_setup=0), upstream_task_ids={'create_fato_email_marketing'}
[2024-08-03T09:53:04.148+0000] {taskinstance.py:2066} INFO - Dependencies not met for <TaskInstance: etl_dag.Carga_Dimensao_user_type scheduled__2024-08-02T15:50:00+00:00 [upstream_failed]>, dependency 'Task Instance State' FAILED: Task is in the 'upstream_failed' state.
[2024-08-03T09:53:04.156+0000] {local_task_job_runner.py:163} INFO - Task is not able to be run
